catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/metatype/org.codice.ddf.catalog.ui.xml

```diff
@@ -46,6 +46,13 @@
             type="String"
             default='[{"type": "stack", "content": [{"type": "component", "component": "cesium", "componentName": "cesium", "title": "3D Map"}, {"type": "component", "component": "inspector", "componentName": "inspector", "title": "Inspector"}]}]'/>
 
+        <AD id="customVisualizations"
+            name="Custom Visualizations"
+            description='Custom Visualizations that should be recognized by golden layout.'
```

Comment:
Can we provide an example in the config description?

Reply:
Actually you bring up a good point, because my values are comma separated, how can I set a default of a list in here? `'"a,b,c","d,e,f"'`?

Reply:
`'"a,b,c,d,e,f"`?

Reply:
Added example and documentation